libogc/network_wii.c

```diff
@@ -931,6 +947,7 @@ s32 net_recv(s32 s, void *mem, s32 len, u32 flags)
 s32 net_recvfrom(s32 s, void *mem, s32 len, u32 flags, struct sockaddr *from, socklen_t *fromlen)
 {
 	s32 ret;
+	const s32 maxblocksize = (NET_HEAP_SIZE-64);
```

Comment:
This is why I'd prefer having this const at global scope so it doesn't need repeating in multiple places.